src/test/java/com/epam/tc/hw2/ex1/FirstExercise.java

```diff
@@ -0,0 +1,135 @@
+package com.epam.tc.hw2.ex1;
+
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class FirstExercise extends TestService {
+
+    @Test
+    public void exerciseOne() {
+
+        SoftAssertions softly = new SoftAssertions();
+
+        //1. Open test site by URL
+        driver.navigate().to(URL);
+
+        //2. Assert Browser title
+        String actualTitle = driver.getTitle();
+        softly.assertThat(actualTitle).isEqualTo(EXPECTED_TITLE);
+
+        //3. Perform login
+        WebElement signIn = driver.findElement(SIGN_IN_BUTTON);
+        signIn.click();
+
+        WebElement login = (new WebDriverWait(driver, 5))
+                .until(ExpectedConditions.visibilityOfElementLocated(LOGIN_INPUT_BUTTON));
+        login.sendKeys(LOGIN_DATA);
+
+        WebElement password = driver.findElement(PASSWORD_INPUT_BUTTON);
+        password.sendKeys(PASSWORD_DATA);
+
+        WebElement enter = driver.findElement(ENTER_BUTTON);
+        enter.click();
+
+        //4. Assert Username is logged in
+        String actualData = driver.findElement(USER_NAME).getText();
+        softly.assertThat(actualData).isEqualTo(EXPECTED_DATA);
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        String actualHomeData = driver.findElement(HOME_BUTTON).getText();
+        softly.assertThat(actualHomeData).isEqualTo(EXPECTED_HOME_DATA);
+
+        boolean isHomeImagePresent = driver.findElement(HOME_BUTTON).isDisplayed();
+        softly.assertThat(isHomeImagePresent).isTrue();
```

Comment:
`isTrue()` could be skipped here

Reply:
<img width="1242" alt="Screenshot 2021-08-08 at 21 45 36" src="https://user-images.githubusercontent.com/84539715/128642580-c88bb8bd-d816-46f3-9d4a-02cd57c46efd.png">

Could you explain, please, why I can skip it? In this case I catch warning (

Reply:
`assertThat` could check boolean expressions by itself. So there is not required call it